src/chrome/content/rules/3_News.xml

```diff
@@ -33,10 +34,10 @@
 	<securecookie host="^(?:www\.)?3news\.co\.nz$" name=".+" />
 
 
-	<rule from="^http://(?:www\.)?3news\.co\.nz/"
-		to="https://www.3news.co.nz/" />
+	<rule from="^https?://(?:www\.)?3news\.co\.nz/"
```

Comment:
We try to avoid redirects from https, except it breaks something in the site (like wrong protocol relative redirects).